### PR TITLE
Added fromBytes to Texture.

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: sdk/src/loom/Graphics/Texture2D.ls [hunk 1]
```diff
@@ -89,6 +89,12 @@ package loom.graphics
          * as a Texture2D.
          */
         public static native function initFromAsset(path:string):TextureInfo;
+        
+        /**
+         * Create a new TextureInfo instance describing the requested asset loaded
+         * as a Texture2D from the provided ByteArray.
+         */
+        public static native function initFromBytes(bytes:ByteArray):TextureInfo;
```
Comment (bengarney) [inline sdk/src/loom/Graphics/Texture2D.ls]: Can you document what format the bytes should be in? (ie, it should be an image file not raw bytes yeah?)

Comment (bengarney) [inline sdk/src/loom/Graphics/Texture2D.ls]: Should we add a way to create it from raw RGBA?

**Comment by bengarney** [comment]
Nitted!

File: sdk/src/loom/Graphics/Texture2D.ls (hunk 1 above)
Comment (SmilyOrg) [inline sdk/src/loom/Graphics/Texture2D.ls]: Good point. Creating a texture from RGBA would be useful too.

**Comment by SmilyOrg** [comment]
Added a couple of fixes.

**Comment by bengarney** [comment]
Do we want to treat texture from RGBA bytes as a followup?

**Comment by SmilyOrg** [comment]
I think so yeah. I put it on my todo.

**Comment by bengarney** [comment]
OK, I'll land once green.

**Pull request merged**